lib/http2/connection.c

```diff
@@ -1307,9 +1311,10 @@ static int foreach_request(h2o_context_t *ctx, int (*cb)(h2o_req_t *req, void *c
     return 0;
 }
 
-void h2o_http2_accept(h2o_accept_ctx_t *ctx, h2o_socket_t *sock, struct timeval connected_at)
+void h2o_http2_accept(h2o_accept_ctx_t *ctx, h2o_socket_t *sock, h2o_iovec_t *http2_origin_frame, struct timeval connected_at)
```

Comment:
Do we need to pass `http2_origin_frame` as a separate argument? To me, using the value in `ctx` seems fine.